Nodejs/Product/Nodejs/Options/NodejsIntellisenseOptionsControl.cs

```diff
@@ -43,7 +43,11 @@ public partial class NodejsIntellisenseOptionsControl : UserControl {
                 if (_fullIntelliSenseRadioButton.Checked) {
                     return AnalysisLevel.High;
                 } else {
-                    return AnalysisLevel.None;
+                    if (_mediumIntelliSenseRadioButton.Checked) {
```

Comment:
Yeah, and you can even go one step further with if, else if, else for best readability.